src/runtime/components/RadioGroup.vue

```diff
@@ -91,17 +93,27 @@ const normalizedItems = computed(() => {
   if (!props.items) return []
   return props.items.map(normalizeItem)
 })
+
+const rootRef = ref()
```

Comment:
What is `rootRef` used for here?

Reply:
This is a leftover from me trying to pass the target element in the change event. I'll clean this up.